app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 30 }
+  validates :description, length: { maximum: 200 }
```

Comment:
meh backend logic should not be related strict to view. You can changes some css to show it in nice way. I will not limit text here, but up to you